relay/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "bonsai-ethereum-relay"
+version = "0.1.0"
+edition = "2021"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[build-dependencies]
+ethers-solc = "1.0.2"
+
+[dependencies]
+anyhow = "1.0"
+async-trait = "0.1.58"
+bincode = "1.3"
+bonsai-sdk-alpha = { package = "bonsai-sdk", git = "https://github.com/risc0/risc0", branch = "release-0.15" }
+bonsai-starter-methods = { path = "../methods" }
+bytemuck = "1.13.1"
+clap = { version = "4.3", features = ["derive", "env"] }
+displaydoc = "0.2"
+ethers = { version = "=2.0.2", features = ["rustls", "ws"] }
+ethers-signers = { version = "2.0.2", features = ["aws"] }
+futures = "0.3"
+hex = "0.4.3"
+pin-project = "1"
+reqwest = { version = "0.11.14", features = ["stream", "json", "gzip"] }
+risc0-build = { git = "https://github.com/risc0/risc0", branch = "release-0.15", features = ["guest-list"] }
+risc0-zkvm = { git = "https://github.com/risc0/risc0.git", branch = "release-0.15", default-features = false }
+rusoto_core = { version = "0.48.0", default-features = false, features = ["rustls"] }
+rusoto_kms = { version = "0.48.0", default-features = false }
+semver = "1.0"
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+snafu = "0.7"
```

Comment:
Seems like there are a lot of dependencies here we don't appear to use. Might be good to do a pass and delete the un-imported ones.